appveyor.yml

```
@@ -16,6 +16,9 @@ environment:
     - nodejs_version: '20'
       PYTHON: "C:\\Python312-x64"
 
+    - nodejs_version: '22'
```

```suggestion
    - nodejs_version: '24'
```

node 24 is the current LTS: https://nodejs.org/en/about/previous-releases